src/main/java/org/spongepowered/api/item/inventory/property/EquipmentSlotType.java

```diff
@@ -66,7 +66,7 @@ public EquipmentSlotType(EquipmentType value, Operator operator) {
      *      properties
      */
     public EquipmentSlotType(Object value, Operator operator) {
-        super(Coerce.<EquipmentType>toPseudoEnum(value, EquipmentType.class, EquipmentTypes.class, EquipmentTypes.WORN), operator);
+        super(Coerce.toPseudoEnum(value, EquipmentType.class, EquipmentTypes.class, EquipmentTypes.WORN), operator);
```

Comment:
No, as in Java 8 implies the generic types in methods, to quote the [Type Inference docs](https://docs.oracle.com/javase/tutorial/java/generics/genTypeInference.html)

>
The compiler requires a value for the type argument T so it starts with the value Object. Consequently, the invocation of Collections.emptyList returns a value of type List<Object>, which is incompatible with the method processStringList. Thus, in Java SE 7, you must specify the value of the value of the type argument as follows:
```
processStringList(Collections.<String>emptyList());
```
This is no longer necessary in Java SE 8. The notion of what is a target type has been expanded to include method arguments, such as the argument to the method processStringList. In this case, processStringList requires an argument of type List<String>. The method Collections.emptyList returns a value of List<T>, so using the target type of List<String>, the compiler infers that the type argument T has a value of String. Thus, in Java SE 8, the following statement compiles:
```
processStringList(Collections.emptyList());
```